recipes/campo/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.3.0" %}
+
+package:
+  name: campo
+  version: {{ version }}
+
+
+source:
+  url: https://campo.computationalgeography.org/download/sources/campo-{{ version }}.tar.bz2
+  sha256: 09c66c68d42dc4fa1ac031f0d52e62b6f1426ff222e3f402c992f4534067aa60
+
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+
+requirements:
+  host:
+    - python  >=3.6
+    - pip
+    - numpy
```

Comment:
Assuming this doesn't contain any compiled code (else it can't be `noarch: python`
```suggestion
```